src/Sql/dbo/Stored Procedures/ProviderOrganizationUnassignedOrganizationDetails_Search.sql

```diff
@@ -0,0 +1,32 @@
+﻿CREATE PROCEDURE [dbo].[ProviderOrganizationUnassignedOrganizationDetails_Search]
+    @Name NVARCHAR(50),
+    @OwnerEmail NVARCHAR(256),
+    @Skip INT = 0,
+    @Take INT = 25
+WITH RECOMPILE
+AS
+BEGIN
+    SET NOCOUNT ON
+    DECLARE @NameLikeSearch NVARCHAR(55) = '%' + @Name + '%'
+
+SELECT
+    O.[Id] AS OrganizationId,
+    O.[Name] AS Name,
+    U.[Email] AS OwnerEmail,
+    O.[PlanType] AS PlanType
+FROM
+    [dbo].[OrganizationView] O
+INNER JOIN
+    [dbo].[OrganizationUser] OU ON O.[Id] = OU.[OrganizationId]
+INNER JOIN
+    [dbo].[User] U ON U.[Id] = OU.[UserId]
```

Comment:
This will return multiple rows if there are multiple Owners. I'm not sure how multiple Owners should be handled, but I assume duplicate rows is probably not the intention. Can you check with Sukhleen?

Reply:
> This will return multiple rows if there are multiple Owners

Excellent observation! I've modified the query to consider that and asked Sukhleen and Priya about removing the owner emails column from the result.

Reply:
Update: We can remove the emails column